pureconfig/src/main/scala/com/avast/sst/pureconfig/WithConfig.scala

```diff
@@ -0,0 +1,17 @@
+package com.avast.sst.pureconfig
+
+import com.typesafe.config.Config
+import pureconfig.ConfigReader
+
+/** Used to retrieve both parsed configuration object and underlying [[Config]] instance.
+  */
+final case class WithConfig[T](value: T, config: Config)
```

Comment:
Please document also usage of this class, thanks.

Reply:
Is this ok? https://github.com/avast/scala-server-toolkit/pull/982/files#diff-f197acdd0685d1ec179f803054d18d6cb451bb451d20901c4f5d33c196a71df0R93